app/lib/ApiRequest/ApiRequest/fetchApiResponseData/index.js

```diff
@@ -15,9 +15,11 @@ const extractTokenFromAuthHeader = (authHeader: ?string): ?string => {
 };
 
 const getDataFromResponse = async (response: Response): Promise<ApiResponseData> => {
-  const responseBody = (response.body)
-    ? await response.json()
+  const responseText = await response.text();
+  const responseBody = (responseText)
+    ? JSON.parse(responseText)
     : {};
```

Comment:
Can we set this to `null` instead of an empty object? That would make it easier to check if the body is empty in the rest of the code, since checking for an empty object is non-trivial.